content/influxdb/v2.0/reference/cli/influx/write/dryrun.md

```diff
@@ -137,7 +137,7 @@ influx write dryrun \
 influx write dryrun \
   --bucket example-bucket \
   --format csv \
-  "#datatype measurement,tag,tag,field,field,ignored,time
+  "#datatype,measurement,tag,tag,field,field,ignored,time
 m,cpu,host,time_steal,usage_user,nothing,time
 cpu,cpu1,host1,0,2.7,a,1482669077000000000
 cpu,cpu1,host2,0,2.2,b,1482669087000000000
```

Comment:
```suggestion
cpu,cpu1,host2,0,2.2,b,1482669087000000000"
```
As is, this breaks when copy-pasted from the source Markdown (but still works when c-p'd from the browser). Close double quote at end of line and delete from line 144. Alternatively, close this line with ` \` and keep `"` on 144.